### PR TITLE
add getStats test

### PR DESCRIPTION
**Pull request by KaptenJansson**
@fippo PTAL

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -30,3 +30,35 @@ test('Check Selenium lib buildDriver method', function(t) {
...
+
+test('Check Selenium lib getStats method', function(t) {
+  var driver = require('../main.js').seleniumLib.buildDriver();
+  var getStats = require('../main.js').seleniumLib.getStats;
+
+  driver.get('file://' + process.cwd() + '/test/testpage.html')
+  .then(function() {
+    t.plan(3);
+    t.pass('Page loaded');
+    return driver.executeScript('window.pc1 = new RTCPeerConnection;' +
```
Comment (fippo) [inline test/test.js]: did you omit the () intentionally? Didn't even know that was possible.

**Comment by fippo** [comment]
needs skip-test for Firefox. LGTM otherwise.
Can you also turn off travis' build on push?

File: test/test.js (hunk 1 above)
Comment (KaptenJansson) [inline test/test.js]: No that was a mistake when cleaning up :(

**Comment by KaptenJansson** [comment]
Turned off build on push, addressed comments (oddly enough `throw 'skip test`' did not work). Also replaced jshint env comments with eslint env. And updated .gitignore + added copyright.

**Comment by fippo** [comment]
:+1:

**Pull request merged**